contrib/babelfishpg_tsql/sql/sys_procedures.sql

```diff
@@ -164,3 +164,19 @@ $$ LANGUAGE plpgsql;
 GRANT EXECUTE ON PROCEDURE sys.sp_babelfish_configure(
 	IN varchar(128), IN varchar(128), IN varchar(128)
 ) TO PUBLIC;
+
+CREATE OR REPLACE PROCEDURE sys.sp_addrole(IN sys.nvarchar(256))
```

Comment:
Input type should be sys.SYSNAME instead of nvarchar. Please double check sql server documentation for these procedures.

Reply:
Checked the documentation and fixed it in next revision